### PR TITLE
4400 replace persona

### PR DESCRIPTION
**Pull request by Bek**

### CODE REVIEW

File: src/encoded/tests/test_auth0.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import pytest
+import requests
+
+@pytest.fixture(scope='session')
+def auth0_access_token():
+    creds = {
+        'connection': 'Username-Password-Authentication',
+        'scope': 'openid',
+        'client_id': 'WIOr638GdDdEGPJmABPhVzMn6SYUIdIH',
```
Comment (hitz) [inline src/encoded/tests/test_auth0.py]: This is a public key, right?

Comment (Bek) [inline src/encoded/tests/test_auth0.py]: Yes

File: src/encoded/static/components/mixins.js [hunk 2]
```diff
@@ -109,6 +108,21 @@ module.exports.Persona = {
             href: window.location.href,
             session_cookie: session_cookie
         });
+        var lock_ = require('auth0-lock');
+        this.lock = new lock_.default('WIOr638GdDdEGPJmABPhVzMn6SYUIdIH', 'encode.auth0.com', {
```
Comment (Bek) [inline src/encoded/static/components/mixins.js]: It creates Auth0Lock object that is the login modal window.

File: buildout.cfg [hunk 3]
```diff
@@ -86,7 +86,7 @@ accession_factory = encoded.server_defaults.test_accession
 file_upload_bucket = encoded-files-dev
 blob_bucket = encoded-blobs-dev
 region_search_instance = region-search-test-v5.instance.encodedcc.org:9200
-indexer_processes = 
+indexer_processes = 16
```
Comment (hitz) [inline buildout.cfg]: remove this file

**Comment by forresttanaka** [comment]
You probably know this already, but just in case: Merge in master, `rm npm-shrinkwrap.json`, then `npm shrinkwrap` and commit the new npm-shrinkwrap.json, that’ll clear that conflict right up.

**Pull request merged**